src/renderers/native/ReactNative.js

```diff
@@ -21,13 +21,15 @@ var ReactUpdates = require('ReactUpdates');
 
 var findNodeHandle = require('findNodeHandle');
 
+import type { ReactElement } from 'ReactElementType';
+
 ReactNativeDefaultInjection.inject();
 
 var render = function(
-  element: ReactElement<any>,
+  element: ReactElement | string | number | null | false,
```

Comment:
FWIW we have a `ReactNode` type definition here: https://github.com/facebook/react/blob/master/src/renderers/shared/fiber/isomorphic/ReactTypes.js#L17 However, as I said, I don't believe this render function accepts a `ReactNode`, only a `ReactElement`.